ember-toucan-core/unpublished-development-types/index.d.ts

```diff
@@ -11,5 +12,6 @@ declare module '@glint/environment-ember-loose/registry' {
     // local entries
     'Form::Field': typeof FieldComponent;
     'Form::Controls::Textarea': typeof TextareaControlComponent;
+    'Form::Controls::Input': typeof InputControlComponent;
```

Comment:
Btw, in our _internal_ types we can also consume our own public registry, so we don't have to add this twice every time. See https://github.com/CrowdStrike/ember-headless-form/blob/main/packages/ember-headless-form/unpublished-development-types/index.d.ts#L32

Reply:
I think @ynotdraw PR is going to get merged first, will pull that from there: https://github.com/CrowdStrike/ember-toucan-core/pull/50

Reply:
Yup! Should be able to rebase and then not have to modify this file! 🎉